### PR TITLE
Add Build and Buildkit Note to Configure CI/CD on Astronomer Software

### LINKED ISSUE #1770: Docker BuildX/Buildkit CI/CD Note
**Affected Document:**

https://docs.astronomer.io/software/ci-cd

**Problem Type (Priority):**

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [x] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**

Add a note that anyone who uses buildkit with buildx needs to add the `--provenance=false` flag

**Additional Notes:**

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1770 

Clarification of where the setting (flag) should be applied is required.

### CODE REVIEW

**Review by rbankston — APPROVED** [review]
LGTM

File: software/ci-cd.md [hunk 1]
```diff
@@ -36,14 +36,20 @@ Automating the deploy process to Astronomer starts with creating a Service Accou
...
+- Authenticate to a Docker Registry
+- Push your image to that Docker Registry
 
 From there, a webhook triggers an update to your Airflow Deployment using the CI/CD tool of your choice. The Astro CLI completes an authentication and push for every manual `$ astro deploy` command.
 
 The rest of this guide describes how to create a service account and what your CI/CD script should look like based on the tool you're using.
 
+:::info
+
+If you're using BuildKit with the Buildx plugin, you need to add the `-provenance=false` setting to your `config.yaml` file.
```
Comment (jwitz) [inline software/ci-cd.md:49]:
```suggestion
If you're using BuildKit with the [Buildx plugin](https://github.com/docker/buildx), you need to add the `--provenance=false` flag to your `docker buildx build` commands.
```
I wasn't able to find reference for a Buildx config.yaml file, nor could I find the `provenance` setting in Astronomer's own `config.yaml` file. Based on the original issue, it seems like the Docker CLI might be the context that we're working with?

Comment (kwcanuck) [reply software/ci-cd.md]: @jwitz any suggestions as to who might be able confirm where this change should be applied?

Comment (jwitz) [reply software/ci-cd.md]: I would tag back @rbankston just to double check that my read is right

Comment (rbankston) [reply software/ci-cd.md]: looks good and the read is right

Comment (kwcanuck) [reply software/ci-cd.md]: @jwitz can you provide your approval so this PR can be merged? Thanks!

**Review by jwitz — APPROVED** [review]

**Pull request merged**